client/jetpack-connect/authorize.js

```diff
@@ -536,12 +535,12 @@ export class JetpackAuthorize extends Component {
 	}
 
 	getRedirectionTarget() {
-		const { clientId, homeUrl, partnerId, redirectAfterAuth } = this.props.authQuery;
+		const { clientId, homeUrl, partnerSlug, redirectAfterAuth } = this.props.authQuery;
 
 		// Redirect sites hosted on Pressable with a partner plan to some URL.
 		if (
 			config.isEnabled( 'jetpack/connect-redirect-pressable-credential-approval' ) &&
-			PRESSABLE_PARTNER_ID === partnerId
+			'pressable' === partnerSlug
```

Comment:
Previously, this would only match `49640` id. Now it will match either 49615 or 49640 pressable ID. Is that OK?

Reply:
Same as above. It should be. 👍